FreeRTOS-Plus/Demo/Common/Mqtt_Demo_Helpers/mqtt_demo_helpers.c

```diff
@@ -55,32 +52,32 @@
 /* Transport interface implementation include header for TLS. */
 #include "using_mbedtls.h"
 
+/* Demo specific config. */
+#include "demo_config.h"
+
 /*------------- Demo configurations -------------------------*/
 
-/** Note: The device client certificate and private key credentials are
- * obtained by the TLS transport interface implementation from the
+/**
+ * Note: The TLS connection credentials for the server root CA certificate,
+ * and device client certificate and private key should be defined in the
  * demo_config.h file.
  */
 
+#ifndef democonfigROOT_CA_PEM
+    #error "Please define the AWS Root CA certificate (democonfigROOT_CA_PEM) in demo_config.h."
+#endif
 #ifndef democonfigCLIENT_PRIVATE_KEY_PEM
-    #error "Please define client private key(democonfigCLIENT_PRIVATE_KEY_PEM) in demo_config.h."
+    #error "Please define client private key (democonfigCLIENT_PRIVATE_KEY_PEM) in demo_config.h."
 #endif
 
 #ifndef democonfigCLIENT_CERTIFICATE_PEM
-    #error "Please define client certificate(democonfigCLIENT_CERTIFICATE_PEM) in demo_config.h."
+    #error "Please define client certificate (democonfigCLIENT_CERTIFICATE_PEM) in demo_config.h."
 #endif
 
 #ifndef democonfigMQTT_BROKER_ENDPOINT
-    #error "Please define democonfigMQTT_BROKER_ENDPOINT in demo_config.h."
+    #error "Please define the AWS IoT broker endpoint (democonfigMQTT_BROKER_ENDPOINT) in demo_config.h."
 #endif
```

Comment:
MQTT demo helpers but I don't see these being used by the coreMQTT demos. These seem specific to AWS because it is configured by mutual-auth and IoT-specific macros.

Reply:
I will rename this macro to be `MQTT_BROKER_ENDPOINT` for now, and will move the demo helpers to under the AWS folder